libs/utils/utils.js

```diff
@@ -155,6 +155,143 @@ function decoratePictures(el) {
   });
 }
 
+export function decorateButtons(el, isLarge) {
```

Comment:
We should not be adding Consonant specific stuff to the generic utils.js file. It's already too big. I would leave things in your decorate.js file.

If we find there are generic pieces that are useful for other non-Consonant blocks, I'm happy to put them in, but I think starting out this way is too heavy handed.